manifests/base/postgres/postgres.yaml

```diff
@@ -82,7 +82,7 @@ spec:
       hostIPC: false
       containers:
         - name: postgresql
-          image: docker.io/bitnami/postgresql:16.2.0-debian-12-r5
+          image: docker.io/bitnami/postgresql:16.3.0-debian-12-r14
```

Comment:
I'm very glad you asked as I had the same thought and had initially gone with `bitnami/postgresql:16.3.0`as it has the same digest SHAs as that very specific one!

Here's the reason I didn't and I'd like to hear what you and maybe @oliverpauffley think about it. Because we use `imagePullPolicy: "IfNotPresent"` the container image will not be downloaded each time. This could lead to a situation where one instance is running say `postgresql:16.3.0-debian-12-r1` and another that starts later is running `postgresql:16.3.0-debian-12-r14` because the patch versions are invisible to you when using `16.3.0`. It just feels like you wouldn't have control of the exact running version which might be confusing if one instance exhibits a bug and another (supposedly on the same version) doesn't.

Does that make any sense?

Reply:
Yeah that makes sense, especially in a shared base!

Reply:
> Yeah that makes sense, especially in a shared base!

Thanks for the sanity check though. I had this exact same thought!